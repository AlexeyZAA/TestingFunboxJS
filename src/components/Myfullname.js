import React, { Component } from 'react';

const Myfullname = ({ firstName, lastName }) => (
<div>
    Автор: { firstName } { lastName }.
</div>
);

export default Myfullname;