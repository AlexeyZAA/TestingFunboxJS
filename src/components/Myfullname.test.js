
import Myfullname from "./Myfullname"

const footer = Myfullname({'firstName': 'Алексей', 'lastName': 'Зубенко'})

it("Тест футера", () => {
    expect(footer).toMatchSnapshot()
});
