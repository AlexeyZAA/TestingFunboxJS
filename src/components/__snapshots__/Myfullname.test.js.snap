// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[object Object] 1`] = `
<div
  class="myfooter"
>
  Автор: 
  <h3>
     
    Алексей
     
    Зубенко
     
  </h3>
  .
</div>
`;

exports[`Тест футера 1`] = `
<div
  class="myfooter"
>
  Автор: 
  <h3>
     
    Алексей
     
    Зубенко
     
  </h3>
  .
</div>
`;
